onnx/defs/traditionalml/defs.cc

```diff
@@ -28,21 +28,43 @@ ONNX_ML_OPERATOR_SET_SCHEMA(
           const auto input_ndim = input_shape.dim_size();
 
           auto output_shape = ctx.getOutputType(0)->mutable_tensor_type()->mutable_shape();
-          // This operator only applies to the last dimension; thus -1
-          for (int i = 0; i < input_ndim - 1; ++i) {
-            *output_shape->add_dim() = input_shape.dim(i);
+          if (input_ndim == 1) {
+            output_shape->add_dim()->set_dim_value(1);
```

Comment:
I don't understand why this is needed. Is the onnxruntime implementation adding an extra dimension in this case?